modules/openapi-generator/src/test/java/org/openapitools/codegen/asciidoc/AsciidocGeneratorTest.java

```diff
@@ -148,4 +148,50 @@ public void testHeaderAttributesFlagRemovesAttributesFromMarkupHeaderSection() t
         Assert.assertTrue(markupFileGenerated, "index.adoc is not generated!");
     }
 
+    @Test
+    public void testGeneratedFilesExist() throws Exception {
+        File output = Files.createTempDirectory("test").toFile();
+
+        final CodegenConfigurator configurator = new CodegenConfigurator().setGeneratorName("asciidoc")
+                .setInputSpec("src/test/resources/3_0/ping.yaml").setOutputDir(output.getAbsolutePath());
+
+        DefaultGenerator generator = new DefaultGenerator();
+        List<File> generatedFiles = generator.opts(configurator.toClientOptInput()).generate();
+
+        Assert.assertFalse(generatedFiles.stream().anyMatch(file -> file.getName().equals("MY-SPEC-DIR/ping.adoc")));
```

Comment:
I don't understand why this assertion is useful. Could you explain the intention please? Thanks!
Also, I think the name of this test doesn't really reflect what the test actually checks.

Reply:
I will change this real quick and the assertion seems to be wrong as well. Will change that, or remove it